.github/workflows/ci.yml

```diff
@@ -4,10 +4,13 @@
 
 name: CI
 
-on: pull_request
+on:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
we tried that in the past. the scheduled run won't be executed if there wasn't a pull request in the past 14 days or something similar. because of that, the scheduled job needs to be in a single file.

Reply:
Hmmm I didn't find any documentation stating that. Maybe a bug of Github Actions?
Maybe it's worth to try it again on this repo that doesn't receive PRs almost at all.

Reply:
It's there in the Foreman org. It showed up in all our repos. I still need to reach out to ask them.

For context, we have:
https://github.com/theforeman/foreman-installer-modulesync/blob/a09d5ed3dc7045230ddf437657d6f9ddf0c7c5a9/moduleroot/.github/workflows/ci.yml.erb#L3-L6

Then we saw that PR testing no longer ran because the cron was disabled.

Reply:
Closing this one then